src/core/client/client.internal.ts

```diff
@@ -77,26 +75,6 @@ export class InternalClient {
       if (profileSettings === undefined) {
         throw profileNotInstalledError(profileId);
       }
-
-      if ('file' in profileSettings) {
-        const filePath = this.fileSystem.path.resolve(
-          this.config.superfacePath,
-          profileSettings.file
-        );
-        if (!(await this.fileSystem.exists(filePath))) {
-          throw profileFileNotFoundError(profileSettings.file, profileId);
-        }
-
-        // TODO: load priority and add it to ProfileConfiguration?
-        const priority = profileSettings.priority;
-        if (!priority.every(p => this.superJson?.providers[p])) {
-          throw unconfiguredProviderInPriorityError(
-            profileId,
-            priority,
-            Object.keys(this.superJson.providers ?? [])
-          );
-        }
-      }
```

Comment:
Priority check should probably stay 🤔 I should specified it in my previous comment.